ls1intum/ArTEMiS
# Feature/migrate modeling from filesystem to database

### Checklist

### Motivation and Context
The UML models and modeling assessments were stored in the file system instead of the database. Therefore, the data model described in https://github.com/ls1intum/ArTEMiS#data-model did not apply to modeling exercises.

### Description
This PR migrates UML models and modeling assessments from the file system to the database. It also changes the data model of modeling assessment to be compliant to the data model described in https://github.com/ls1intum/ArTEMiS#data-model, i.e. modeling assessments are represented by a Feedback List contained in a Result for the corresponding modeling submission.

### Steps for Testing
1. Log in to ArTEMiS as a student
2. Navigate to the course overview
3. Save/Submit a submission for a modeling exercise
4. Check database for stored model of the submitted modeling submission
5. Log in to ArTEMiS as a tutor
6. Navigate to assessment dashboard
7. Submit an assessment for the previously submitted submission
8. Check database for stored Result and Feedback entries
9. In general, play around with modeling submissions and assessments (also automatic assessments) and check, if everything works as expected.
